<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingParameter</class>
 <widget class="QWidget" name="SettingParameter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="l_value">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>39</y>
     <width>45</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Cond Bold</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>38</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-30</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLabelLongPressAnimated" name="l_up">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>37</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/flecha_der.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="MyLabelLongPressAnimated" name="l_down">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>37</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/flecha_izq.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="l_value_min">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>117</y>
     <width>45</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Cond Bold</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLabelLongPressAnimated" name="l_up_min">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>117</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/flecha_der.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="MyLabelLongPressAnimated" name="l_down_min">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>117</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/flecha_izq.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="l_value_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>122</y>
     <width>64</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Cond Bold</family>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 222, 99);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/peep_label.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_value_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>42</y>
     <width>64</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Cond Bold</family>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 222, 99);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/pmax_label_2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLabelAnimated" name="l_ok">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>327</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Cond Bold</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
     border: 2px solid #E3E3E3;
     border-radius: 5px;
     color: #E3E3E3
 }</string>
   </property>
   <property name="text">
    <string>ACEPTAR</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_value_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>3</width>
     <height>360</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Cond Bold</family>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>PEEK</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_value_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Cond Bold</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/cmH2O.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_value_6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Cond Bold</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/cmH2O.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLabelLongPressAnimated" name="l_up_fio2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/flecha_der.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="l_value_fio2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>45</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Cond Bold</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLabelLongPressAnimated" name="l_down_fio2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/flecha_izq.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="l_value_7">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>201</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Cond Bold</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/porciento.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_value_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>199</y>
     <width>64</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 222, 99);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/FiO2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="widget_capnography" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <widget class="MyLabelLongPressAnimated" name="l_down_etco2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="assets.qrc">:/icons/flecha_izq.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="l_value_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>64</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DIN Pro</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 222, 99);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="assets.qrc">:/icons/PCO2.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_value_10">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>13</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DIN Pro Cond Bold</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="assets.qrc">:/icons/mmHg.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="MyLabelLongPressAnimated" name="l_up_etco2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="assets.qrc">:/icons/flecha_der.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="l_value_etco2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>45</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DIN Pro Cond Bold</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <zorder>l_value</zorder>
  <zorder>l_title</zorder>
  <zorder>l_value_min</zorder>
  <zorder>l_up_min</zorder>
  <zorder>l_down_min</zorder>
  <zorder>l_down</zorder>
  <zorder>l_up</zorder>
  <zorder>l_value_2</zorder>
  <zorder>l_value_3</zorder>
  <zorder>l_ok</zorder>
  <zorder>l_value_4</zorder>
  <zorder>l_value_5</zorder>
  <zorder>l_value_6</zorder>
  <zorder>l_up_fio2</zorder>
  <zorder>l_value_fio2</zorder>
  <zorder>l_down_fio2</zorder>
  <zorder>l_value_7</zorder>
  <zorder>l_value_8</zorder>
  <zorder>widget_capnography</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabelAnimated</class>
   <extends>QLabel</extends>
   <header location="global">mylabelanimated.h</header>
  </customwidget>
  <customwidget>
   <class>MyLabelLongPressAnimated</class>
   <extends>QLabel</extends>
   <header location="global">mylabellongpressanimated.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
